ipatests/test_integration/test_commands.py

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+#
+
+from tempfile import NamedTemporaryFile
+import textwrap
+
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.pytest_plugins.integration import tasks
+
+
+class TestCommands(IntegrationTest):
+    """
+    A lot of commands can be executed against a single IPA installation
+    so provide a generic class to execute one-off commands that need to be
+    tested without having to fire up a full server to run one command.
```

Comment:
I like the idea! :+1: 

I even like the idea so much, that I fired up my time machine and added ``test_ipa_cli`` in the past. :) But your file name better. Please merge and fold ``test_ipa_cli`` content into your new file and update ``.freeipa-pr-ci.yaml``.

Reply:
Done.